python/sdss_access/sync/baseaccess.py

```diff
@@ -35,7 +35,8 @@ def remote(self, username=None, password=None, inquire=None):
         use_dtn = self.remote_scheme == 'rsync'
         # simplifies things to have a single sdss (or sdss5) machine in
         # .netrc for SDSS-IV  (or SDSS-V, respectively).
-        sdss5 = ( self.release in self._s5cfgs )
+        #sdss5 = ( self.release in self._s5cfgs )
```

Comment:
yes, it can be removed and a comment added such as:
```
# set sdss5 true if s5cfg (e.g. sdsswork or ipl-N) starts with
# values defined in self._s5cfgs (e.g. sdss or ipl).
```